DESCRIPTION

```diff
@@ -1,6 +1,5 @@
 Package: LPWC
-Version: 0.99.5
-Date: 2019-03-12
+Version: 1.0.0
```

Comment:
Please update `NEWS.md` with the changes in this release

The reference in the `Description` line below also needs to be updated.

Reply:
Done.